annualreviews/2022-ServerlessWorkflowSpecification-annual.md

```diff
@@ -0,0 +1,152 @@
+
+Project [DevStats page](https://serverlessworkflow.devstats.cncf.io).
+The info for the span of one year shows:
+
+* Over 50 new forks
+<!-- 
+We need this updated data, I couldn't read this devstats page, so confusing!
+* Over 500 merged PRs 
+* ~2000 commits by 30+ contributors
+* Community contributions from 10+ different companies
+-->
+* Over 101% increase to github stars compared to last year
+
+- **How many maintainers do you have, and which organizations are they from?**
+
+Serverless Workflow currently has [3 project maintainers](https://github.com/serverlessworkflow/specification/blob/main/MAINTAINERS.md)
+
+<!-- Alphabetical order -->
+- Charles d'Avernas, Neuroglia
+- Ricardo Zanini, Red Hat
+- Tihomir Surdilovic, Temporal Technologies
+
+- **What do you know about adoption, and how has this changed since your last review / since you joined Sandbox?**  
+
+Both the adoption and community interest has been steadily increasing over the course of the year.
+Most notable adoptions have been by:
+    - [Kogito](https://kogito.kie.org/), a Red Hat project automation runtime
+    - [OpenShift Serverless Logic](https://developers.redhat.com/articles/2022/08/15/how-openshift-serverless-logic-evolved-improve-workflows), a Red Hat product under Tech Preview integrated with their flagship product, OpenShift
+    - [Automatiko](https://automatiko.io/), a workflow automation runtime
+    - [Synapse](https://github.com/serverlessworkflow/synapse), a Kubernetes-based workflow runtime which has joined the Serverless Workflow ecosystem
+    - [Apache EventMesh](https://eventmesh.apache.org/), a new generation serverless event middleware for building distributed event-driven applications
+    <!-- I believe we have others, can we link them here? -->
+
+We also have a number of integrations that are currently work-in-progress which include
+integrations with [Temporal](https://temporal.io/).
+
+- **How has the project performed against its goals since the last review?**
+
+This is our second annual review since becoming a Sandbox project.
+This year we looked for stabilization of the specification by having more
+discussions with current implementation project leaders to achieve a good
+balance between the standards and production-level use cases.
+
+We decided to grow slowly and now we are reaching to a point to fix most of
+the open issues and discussions before releasing 0.9 version, and then the final
+1.0 by the end of the year.
+
+One of the goals we achieved, was to set a new governance model to balance the
+responsibilities amongst all the maintainers.
+
+We had to do a little detour duo to the progress of projects implementing and using
+the specification in many production-level use cases.
+
+- **What are the current goals of the project?**
+
+Specification roadmap: https://github.com/serverlessworkflow/specification/tree/main/roadmap
```

Comment:
I think we can also include this: https://github.com/orgs/serverlessworkflow/projects/1/views/2